filament/src/MaterialInstance.cpp

```diff
@@ -58,6 +61,11 @@ FMaterialInstance::FMaterialInstance(FEngine& engine, FMaterial const* material)
         static_cast<MaterialInstance*>(this)->setParameter(
                 "maskThreshold", material->getMaskThreshold());
     }
+
+    if (material->hasDoubleSidedCapability()) {
+        static_cast<MaterialInstance*>(this)->setParameter(
+                "doubleSided", material->isDoubleSided());
```

Comment:
We should use a prefix on those built-in parameters. Maybe _ or filament_. Mask threshold should too.

Reply:
Done, pushed a new commit for this.